Rarevariantburden test datasets

saima-tithi-stjude opened the pull request.

maxulysse commented:
It looks good, but if I may some recommendation.
You already have the data in a `rarevariantburden` branch I do not think there's any need for a `rarevariantburden` folder in the architecture of the files.
I would also recommend a README where you explain the files, what they are for, etc.

saima-tithi-stjude commented:
Hi Maxime, I have updated the README file with description of the test files, and also deleted rarevariantburden folder from the folder structure. Could you please check it now?

maxulysse commented:
> Hi Maxime, I have updated the README file with description of the test files, and also deleted rarevariantburden folder from the folder structure. Could you please check it now?

Looking super good to me

maxulysse commented:
Feel free to merge
